lucille-core/src/test/java/com/kmwllc/lucille/indexer/NoOpIndexer.java

```diff
@@ -0,0 +1,27 @@
+package com.kmwllc.lucille.indexer;
+
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Indexer;
+import com.kmwllc.lucille.message.IndexerMessenger;
+import com.typesafe.config.Config;
+import java.util.List;
+
+public class NoOpIndexer extends Indexer {
```

Comment:
I'm actually thinking we should put this in the main tree as opposed to test
there are enough cases where someone wants to run an ingest without actually sending to a search engine
we say to configure a regular indexer and set sendEnabled=false, but I'm coming around to thinking a NoOpIndexer might be a better way. We have a stage in the test tree called NoopStage without second o in caps, should probably be consistent and call this NoopIndexer. I think I actually prefer NopIndexer and NopStage -- thoughts?